tasks/src/depth-estimation/about.md

```diff
@@ -0,0 +1,38 @@
+## Use Cases
+Depth estimation models can be used to approximate the depth of different objects present in an image.
+
+### Estimation of Volumetric Information
+Depth estimation models are widely used to study volumetric formation of objects present inside an image. This is an
+important use case in the domain of computer graphics. 
+
+### 3D Representation
+
+Models trained for depth estimation can also be used to develop a 3D understanding from a 2D image which is another
+important use case for computer graphics.
+
+## Inference
+
+With the `transformers` library, you can use the `depth-estimation` pipeline to infer with image classification models. You can initialize the pipeline with a model id from the Hub. If you do not provide a model id it will initialize with [Intel/dpt-large](https://huggingface.co/Intel/dpt-large) by default. When calling the pipeline you just need to specify a path, http link or an image loaded in PIL. 
+
+```python
+from transformers import pipeline
+
+estimator = pipeline("depth-estimation")
+result = estimator("http://images.cocodataset.org/val2017/000000039769.jpg")
+result
+
+# {'predicted_depth': tensor([[[ 6.3199,  6.3629,  6.4148,  ..., 10.4104, 10.5109, 10.3847],
```

Comment:
Would be nice to add a line below to visualize the depth image like you did in schema.

Reply:
Added a comment.